testsuite/validation/validation/pot3d.out

```diff
@@ -440,6 +440,6 @@
  ### Magnetic energy diagnostic:
 
 Energy in Br**2 = 4.19882181E+01
-Energy in Bt**2 = 1.26745493E+01
+Energy in Bt**2 = 1.26748493E+01
```

Comment:
the script for *validating* the output here: https://github.com/predsci/POT3D/blob/main/scripts/pot3d_validation.sh, only compares three values in the output file (see sample output file here: https://github.com/predsci/POT3D/blob/main/testsuite/validation/validation/pot3d.out), only compares the values of three lines i.e.

"Energy in Br"
"Energy in Bt"
"Energy in Bp"

Now, the question is whether comparing only these three value suffices for *validation* or not.